core/arch/arm/plat-ti/conf.mk

```diff
@@ -15,6 +15,7 @@ $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_SECURE_TIME_SOURCE_REE,y)
 arm32-platform-cpuarch		:= cortex-a9
 else
+CFG_OTP_SUPPORT ?= y
```

Comment:
Sorry for changing my mind ;) but in fact I think this should just go. `CFG_OTP_SUPPORT` does not appear anywhere else now, and you already have the `plat_huk[]` array populated unconditionally, so it is a bit weird not to use it. Let's make things as simple as possible. OK?